website/docs/validating-admission-policy.md

```diff
@@ -108,13 +108,14 @@ spec:
 
 With this new engine and source added to the constraint template, now Gatekeeper webhook, audit, and shift-left can validate resources with these new CEL-based rules.
 
-## Policy updates to generate Validating Admission Policy resources
+## Policy updates to generate Validating Admission Policy and Binding resources
 
 For some policies, you may want admission requests to be handled by the K8s Validating Admission Controller instead of the Gatekeeper admission webhook.
 
-Gatekeeper is configured to generate K8s Validating Admission Policy (VAP) resources for all constraint templates globally if `--default-create-vap-for-templates=true` flag is set. This flag defaults to `false` at this time to not generate VAP resources by default.
+The K8s Validating Admission Controller requires both the Validating Admission Policy (VAP) and Validating Admission Policy Binding (VAPB) resources to exist to enforce a policy. Gatekeeper can be configured to generate both of these resources. To generate VAP Bindings for all Constraints, ensure the Gatekeeper 
+`--default-create-vap-binding-for-constraint` flag is set to `true`. To generate VAP as part of all Constraint Templates with cel engine, ensure the Gatekeeper `--default-create-vap-for-templates=true` flag is set to `true`. By default both flags are set to `false` while the feature is still in alpha.
```

Comment:
"all Constraint Templates that have VAP CEL code" (or use the actual engine name, IIRC `K8sNativeValidation`?)

We don't want to conflate CEL with VAP, because that will be a common, and very misleading, mistake. It assumes that VAP code can be accepted anywhere CEL is interpreted, which is very much not true.

Reply:
added VAP to `CEL engine` where applicable